gists/remote-state-upgrade/errors.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import "errors"
+
+var (
+	ErrNotFound         = errors.New("okctl/NotFound")
+	ErrAlreadyExists    = errors.New("okctl/AlreadyExists")
+	ErrNotAuthenticated = errors.New("okctl/NotAuthenticated")
+	ErrBadRequest       = errors.New("okctl/BadRequest")
+)
+
+const (
+	BadCurrentWorkingDirectory = iota
+	BranchDirty
+	BranchInvalid
+	BranchOutdated
+	BucketAlreadyExists
+	ClusterManifestNotFound
+	NotAuthenticated
+)
+
+func t(key int) string {
+	switch key {
+	case ClusterManifestNotFound:
+		return "Cluster manifest not found at provided path, are you sure it is correct?"
+	case BucketAlreadyExists:
+		return "The bucket already exists. To not overwrite any important data, please contact #kjøremiljø"
```

Comment:
Clarifying language a bit, should probably say:
"To avoid overwriting any important data"